### DISCUSSION TITLE
Can't download image from URL

### DISCUSSION THREAD

**Discussion by abdulrehmandar2234** [spatie/laravel-medialibrary#2387] [Q&A]
addMediaFromUrl() is working fine but when I tried to download image from this url

https://cdn.shopify.com/s/files/1/0378/5231/5780/products/6991391_360x.jpg

Error: Spatie\MediaLibrary\MediaCollections\Exceptions\UnreachableUrl
Url `http://cdn.shopify.com/s/files/1/0378/5231/5780/products/6991391_360x.jpg` cannot be reached

I don't think its URL problem because get_file_content() can download this file.

And this one is not extracting the image may be this url contain variables
 
https://media.continente.pt/Sonae.eGlobal.Presentation.Web.Media/media.axd?resourceSearchType=2&resource=ProductId=4071749(eCsf$RetekProductCatalog$MegastoreContinenteOnline$Continente)&siteId=1&channelId=1&width=150&height=150&defaultOptions=1

**Comment by typoworx-de** [comment]
Also having issues with addMediaFromUrl with https. I'm using PHP 8.0 if this is useful for solving.

**Reply by typoworx-de** [reply]
Don't know why, but for some reason PHp didn't recognice system ca-certs.

Adding this to php.ini (or included "mods-available/openssl.ini") worked:
openssl.cafile=/etc/ssl/certs/ca-certificates.crt
